docs/content/sftp.md

```diff
@@ -149,7 +149,7 @@ Modified times are used in syncing and are fully supported.
 
 ### Limitations ###
 
-SFTP does not support any checksums.
+SFTP supports checksums if the same login has shell access and md5sum or sha1sum are in the remote's PATH.
```

Comment:
You could put a note in docs/content/overview.md too - probably yet another asterisk note about it.